modules/local/modkit.nf

```diff
@@ -0,0 +1,54 @@
+
+process MODKIT {
+
+    tag "$meta.id"
+    label 'process_high'
+    container "ghcr.io/eitm-org/modkit"
+
+    publishDir "$params.outdir/modkit", mode: params.publish_dir_mode
+    // note: publishDir info could be moved to conf/modules/modules.config
+
+    input:
+
+    tuple val(meta), path(bam), path(bai)
+    path fasta
+
+    output:
+
+    path('*.bed')
+    path(summary)
+    path(modkit_log)
+    path("versions.yml"), emit: versions
+
+    script:
+
+    prefix = task.ext.prefix ?: "${meta.id}"
+    bed = "${prefix}.bed"
+    summary = "${prefix}.summary"
+    modkit_log = "${prefix}.log"
+
+    // modkit command line options:
+    //  https://nanoporetech.github.io/modkit/
+
+    modkit_args = " --ref $fasta"
```

Comment:
ext.args from modkit.config should be passed into here right? Example https://github.com/eitm-org/sarek/blob/master/modules/local/clairs.nf#L32 and https://github.com/eitm-org/sarek/blob/master/conf/modules/clairs.config#L50